Source/santad/SNTExecutionController.mm

```diff
@@ -265,15 +265,15 @@ - (void)validateExecEvent:(const Message &)esMsg postAction:(bool (^)(SNTAction)
       absl::ReaderMutexLock lock(&self->_entitlementFilterMutex);
 
       if (teamID && self->_entitlementsTeamIDFilter.count(std::string(teamID)) > 0) {
-        LOGD(@"Dropping entitlement logging for configured TeamID: %s", teamID);
+        // Dropping entitlement logging for configured TeamID
```

Comment:
Unrelated changes?

Reply:
Yeah, basically just didn't want these debug logs anymore since they're kinda noisy (and didn't feel like doing a whole PR cycle for just removing the debug prints).